tests/support.cpp

```diff
@@ -82,4 +82,121 @@ TEST(DriveIndex, DriveZ)
 	EXPECT_EQ(25, drive_index('Z'));
 }
 
+TEST(Support_split_delim, NoBoundingDelims)
```

Comment:
I though I made this comment earlier but I can't find it now…

An edge-case test is missing: splitting a non-empty string that has no characters for splitting, e.g.

`split("abc", 'x') -> {"abc"}`, `split("abc") -> {"abc"}`

Reply:
(Good news is that the your prior comment showed up on my side and wasn't lost)